source/_components/solarlog.markdown

```diff
@@ -0,0 +1,107 @@
+---
+title: "Solar-Log Sensor"
+description: "Instructions on how to integrate Solar-Log sensors within Home Assistant."
+logo: solar-log.png
+ha_category: Sensor
+ha_release: 0.100
```

Comment:
```suggestion
ha_release: "0.100"
```

Reply:
Fixed, I also added double quotes around the IoT class